### PR TITLE
feat- Button documentation

### PR DESCRIPTION
**Pull request by Ayesha-7**
added side-nav
added doc for button
image

~not responsive yet

### CODE REVIEW

**Review by sharathnair9999 — APPROVED** [review]

**Review by sharathnair9999 — APPROVED** [review]

File: Styles/docs.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.side-nav-link:hover {
+  background: #0084ff34;
+  border-radius: 6px;
+  padding-left: 24px;
+  font-weight: bold;
+  cursor: pointer;
+}
+
```
Comment (Richa-rani02) [inline Styles/docs.css]: suggestion: u can define variable for colors here also

File: index.html [hunk 2]
```diff
@@ -86,9 +94,21 @@ <h3 class="m-1 text bold">Lorem ipsum dolor sit amet.</h3>
...
-          <li><i class="fa fa-twitter mx-1" style="font-size: 28px;"></i></li>
+          <li>
+            <a href="https://github.com/Ayesha-7/" target="blank"
+              ><i class="fa fa-github mx-1" style="font-size: 28px;"></i
+            ></a>
+          </li>
+          <li>
+            <a href=" www.linkedin.com/in/ayeshashaikh7" target="blank"
+              ><i class="fa fa-linkedin mx-1" style="font-size: 28px;"></i
+            ></a>
```
Comment (Richa-rani02) [inline index.html]: suggestion: don't use inline css

**Review by tewarimohit — APPROVED** [review]
Looks good to merge...

File: index.html [hunk 3]
```diff
@@ -86,9 +94,21 @@ <h3 class="m-1 text bold">Lorem ipsum dolor sit amet.</h3>
...
+          </li>
+          <li>
+            <a href="https://twitter.com/_ayesha_7" target="blank"
+              ><i class="fa fa-twitter mx-1" style="font-size: 28px;"></i
+            ></a>
```
Comment (Richa-rani02) [inline index.html]: here also

File: Docs/button.html [hunk 4]
```diff
@@ -4,9 +4,286 @@
...
+          <li>
+            <a href=" www.linkedin.com/in/ayeshashaikh7" target="blank"
+              ><i class="fa fa-linkedin mx-1" style="font-size: 28px;"></i
+            ></a>
+          </li>
+          <li>
+            <a href="https://twitter.com/_ayesha_7" target="blank"
+              ><i class="fa fa-twitter mx-1" style="font-size: 28px;"></i
+            ></a>
+          </li>
```
Comment (Richa-rani02) [inline Docs/button.html]: here also

File: Docs/button.html [hunk 5]
```diff
@@ -4,9 +4,286 @@
...
+            <a href="https://github.com/Ayesha-7/" target="blank"
+              ><i class="fa fa-github mx-1" style="font-size: 28px;"></i
+            ></a>
+          </li>
+          <li>
+            <a href=" www.linkedin.com/in/ayeshashaikh7" target="blank"
+              ><i class="fa fa-linkedin mx-1" style="font-size: 28px;"></i
+            ></a>
+          </li>
+          <li>
```
Comment (Richa-rani02) [inline Docs/button.html]: here

File: Docs/button.html [hunk 6]
```diff
@@ -4,9 +4,286 @@
...
+              Use classes as shown below in example and make Icon buttons.
+            </p>
+          </div>
+          <div class="component-container my-2 p-2">
+            <button class="btn btn-icon">
+              <img src="../assets/images/heart.png" />
+            </button>
+            <button class="btn btn-icon-label">
+              <img src="../assets/images/share.png" />
+              <div>Share</div>
```
Comment (Richa-rani02) [inline Docs/button.html]: suggestion: use alt with img

File: Docs/button.html [hunk 7]
```diff
@@ -4,9 +4,286 @@
...
+              ></a>
+            </h6>
+          </li>
+          <li class="main-nav-link px-1">
+            <h6 class="text-bold">
+              <a href="https://github.com/Ayesha-7/Component-Library"
+                >Github<i
+                  class="fa fa-github px-05"
+                  style="font-size: 24px;"
+                ></i>
```
Comment (Richa-rani02) [inline Docs/button.html]: here also dont use inline styles

**Review by Richa-rani02 — CHANGES_REQUESTED** [review]

File: Docs/button.html (hunk 7 above)
Comment (Ayesha-7) [reply Docs/button.html]: thanks for pointing out Richa, removed all inline styles.

File: Styles/docs.css (hunk 1 above)
Comment (Ayesha-7) [reply Styles/docs.css]: thanks for the suggestion! updated the code used variable for color.

File: Docs/button.html (hunk 6 above)
Comment (Ayesha-7) [reply Docs/button.html]: added alt attribute to all img tags

**Review by Richa-rani02 — APPROVED** [review]

File: index.html [hunk 8]
```diff
@@ -35,17 +35,18 @@ <h2 class="text-bolder">ace.</h2>
...
+                >Docs<i class="fa fa-file fa-3 px-05"></i
               ></a>
             </h6>
           </li>
           <li class="main-nav-link px-1">
             <h6 class="text-bold">
-              <a href="https://github.com/Ayesha-7/Component-Library"
-                >Github<i
-                  class="fa fa-github px-05"
-                  style="font-size: 24px;"
```
Comment (deepak29-git) [inline index.html:47]: avoid inline style

File: index.html [hunk 9]
```diff
@@ -86,9 +94,21 @@ <h3 class="m-1 text bold">Lorem ipsum dolor sit amet.</h3>
     <footer class="footer-container flex-column">
       <div class="footer-social mx-05 mt-05">
         <ul class="footer-social-links flex-row">
-          <li><i class="fa fa-github mx-1" style="font-size: 28px;"></i></li>
```
Comment (deepak29-git) [inline index.html:89]: same for here (avoid inline style)

**Review by deepak29-git — COMMENTED** [review]
Rest looking good to me

**Pull request merged**